.github/workflows/zips.yml

```diff
@@ -208,3 +214,27 @@ jobs:
           comment-id: ${{ steps.find-comment.outputs.comment-id }}
           edit-mode: replace
           body: Build files for ${{ github.event.pull_request.head.sha }} have been deleted.
+
+  publish-to-wporg:
+    name: Publish to WordPress.org
+    runs-on: ubuntu-latest
+    environment: wordpress-plugin-svn
+    if: github.event_name == 'workflow_dispatch' && ( github.ref_type == 'tag' || inputs.release_version != '' )
+    needs: build-zips
+    timeout-minutes: 20
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: zip-files
+          path: ${{ github.ref }}
+      - name: Extract
+        run: |
+          unzip google-site-kit.zip
+          mv google-site-kit/* .
+          rm -rf google-site-kit *.zip
+          echo .distignore > .distignore
+      - uses: 10up/action-wordpress-plugin-deploy@stable
```

Comment:
What do you think about pinning this action's version to a hash to prevent unexpected changes? I realize we don't do this for other actions, but this a pretty critical one.

Reply:
Updated to use the latest version `2.2.2`.